receiver/mongodbreceiver/testdata/serverStatus.json

```diff
@@ -0,0 +1,3628 @@
+{
```

Comment:
This is massive. Is there a way to instruct the server to return less (closer to what we actually will consume)?

If not, at least consider trimming down this example to represent a smaller cluster?